src/rust/rust-proto-new/src/graplinc/grapl/api/plugin_work_queue/v1beta1.rs

```diff
@@ -252,17 +252,13 @@ impl TryFrom<proto::GetExecuteAnalyzerResponse> for GetExecuteAnalyzerResponse {
     fn try_from(value: proto::GetExecuteAnalyzerResponse) -> Result<Self, Self::Error> {
         let request_id = value.request_id;
         let maybe_job = value.maybe_job.ok_or(Self::Error::MissingField(
-            "GetExecuteAnalyzerResponse.execution_job",
+            "GetExecuteAnalyzerResponse.maybe_job",
         ))?;
         let execution_job: Option<ExecutionJob> = maybe_job.try_into()?;
 
```

Comment:
Turns out our serde for GetExecute*Response was incorrect - it's completely valid to have a None maybe_job.